src/main/java/tasks/Task.java

```diff
@@ -0,0 +1,83 @@
+package tasks;
+/**
+* Task class to create a task for the user to do.
+*/
+public class Task {
+    protected boolean isDone = false;
+    protected String taskName = "";
+
+    ////////////////////////////////////////////////////////////////
+    // Constructors
```

Comment:
If you're using IntelliJ IDEA (not sure about other IDEs/text editors), there's a feature where you can mark out code regions using the following syntax:

```
//region Constructors

...

//endregion
```

It can allow code folding and overall looks cleaner as well.